src/main/java/org/openrewrite/java/migrate/javax/AutoGeneratedValue.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2024 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.migrate.javax;
+
+import lombok.EqualsAndHashCode;
+import lombok.Value;
+import org.openrewrite.*;
+import org.openrewrite.java.*;
+import org.openrewrite.java.search.FindAnnotations;
+import org.openrewrite.java.search.UsesType;
+import org.openrewrite.java.tree.*;
+
+import java.util.Comparator;
+import java.util.List;
+import java.util.Set;
+
+@Value
+@EqualsAndHashCode(callSuper = false)
+public class AutoGeneratedValue extends Recipe {
+    @Override
+    public String getDisplayName() {
+        return "Attributes with automatically generated values require configuration.";
+    }
+
+    @Override
+    public String getDescription() {
+        return "Adds missing configuration for attributes with automatically generated values.";
```

Comment:
I think we can do more with markdown here to describe the changes made to the user. Right now if I'd search for `GeneratedValue` in the docs I'd not have any hits using these descriptions. I know you sometimes refer to other sources for a description of the changes, but perhaps we can be more descriptive here in the description?